tests/credential_plugins_test.py

```diff
@@ -162,3 +165,44 @@ def test_tss_import(self) -> None:
         ):
             # assert this module as opposed to older thycotic.secrets.server
             assert cls.__module__ == 'delinea.secrets.server'
+
+
+def test_aim_sensitive_traceback_masked(
+        mocker: MockerFixture,
+        monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """Ensure that the sensitive information is not leaked in the traceback."""
+    my_response = requests.Response()
+    my_response.status_code = 404
+    my_response.url = 'not_found'
+
+    mock_get = mocker.patch(
```

Comment:
why didn't you keep monkeypatch?

Reply:
not sure, lemme go see. I wonder if something happened in a conflict merge.

Reply:
Marking this as resolved.